vault/dendron.ref.commands.md

```diff
@@ -28,6 +28,13 @@ nav_order: 3.1
 
 Open the GitHub repository (if it exists) of the current note. If you have a [[multi vault|dendron.topic.multi-vault]] workspace with different git repositories, this command will
 
+### Copy Codespace URL
+
+- shortcuts: none
+
+Copy the Codespace URL (if it exists) of the current note.
```

Comment:
Can we briefly explain here when this command will work? i.e., they must have a github backed vault.